packages/fluentui/react-northstar/src/components/Toolbar/ToolbarMenu.tsx

```diff
@@ -73,103 +73,129 @@ export interface ToolbarMenuProps extends UIComponentProps, ChildrenComponentPro
+  (props, ref, composeOptions) => {
+    const context: ProviderContextPrepared = React.useContext(ThemeContext);
+    const { setStart, setEnd } = useTelemetry(composeOptions.displayName, context.telemetry);
+    setStart();
+
+    const { accessibility, className, children, design, items, submenu, submenuIndicator, styles, variables } = props;
+
+    const parentVariables = React.useContext(ToolbarVariablesContext);
+    const mergedVariables = mergeComponentVariables(parentVariables, variables);
+
+    const getA11yProps = useAccessibility(accessibility, {
+      debugName: composeOptions.displayName,
+      actionHandlers: {
+        performClick: e => {
+          _.invoke(props, 'onClick', e, props);
+        },
+      },
+      rtl: context.rtl,
+    });
+    const { classes } = useStyles<ToolbarMenuStylesProps>(composeOptions.displayName, {
+      className: composeOptions.className,
+      composeOptions,
+      mapPropsToInlineStyles: () => ({
+        className,
+        design,
+        styles,
+        variables: mergedVariables,
+      }),
+      rtl: context.rtl,
+      unstable_props: props,
+    });
 
-  const parentVariables = React.useContext(ToolbarVariablesContext);
-  const mergedVariables = mergeComponentVariables(parentVariables, variables);
+    const handleItemOverrides = predefinedProps => ({
+      onClick: (e, itemProps) => {
+        _.invoke(predefinedProps, 'onClick', e, itemProps);
+        _.invoke(props, 'onItemClick', e, {
+          ...itemProps,
+          menuOpen: !!itemProps.menu,
+        });
+      },
+    });
 
-  const getA11yProps = useAccessibility(accessibility, {
-    debugName: ToolbarMenu.displayName,
-    actionHandlers: {
-      performClick: e => {
-        _.invoke(props, 'onClick', e, props);
+    const handleRadioGroupOverrides = (predefinedProps: ToolbarMenuRadioGroupProps) => ({
+      onItemClick: (e, itemProps) => {
+        _.invoke(predefinedProps, 'onItemClick', e, itemProps);
+        _.invoke(props, 'onItemClick', e, itemProps);
       },
-    },
-    rtl: context.rtl,
-  });
-  const { classes } = useStyles<ToolbarMenuStylesProps>(ToolbarMenu.displayName, {
-    className: toolbarMenuClassName,
-    mapPropsToInlineStyles: () => ({
-      className,
-      design,
-      styles,
-      variables: mergedVariables,
-    }),
-    rtl: context.rtl,
-  });
-
-  const handleItemOverrides = predefinedProps => ({
-    onClick: (e, itemProps) => {
-      _.invoke(predefinedProps, 'onClick', e, itemProps);
-      _.invoke(props, 'onItemClick', e, {
-        ...itemProps,
-        menuOpen: !!itemProps.menu,
+    });
+
+    const renderItems = () => {
+      return _.map(items, item => {
+        const kind = _.get(item, 'kind', 'item');
+
+        switch (kind) {
+          case 'divider':
+            return createShorthand(ToolbarMenuDivider, item);
+
+          case 'group':
+            return createShorthand(ToolbarMenuRadioGroup, item, { overrideProps: handleRadioGroupOverrides });
```

Comment:
```suggestion
            return createShorthand(composeOptions.slots.group, item, { overrideProps: handleRadioGroupOverrides });
```

or even `composeOptions.slots.radioGroup`, I suggested group as the kind has that value